src/storage/src/hummock/sstable/block.rs

```diff
@@ -177,6 +178,26 @@ impl Debug for Block {
     }
 }
 
+impl Serialize for Block {
```

Comment:
Is `Serialize` and `Deserialize` intended only for temporary caching rather than for persistent storage? We can add some doc or ref

Reply:
They are for disk cache only.

foyer receives these trait for in-memory cache or disk cache:

```rust
pub trait Key: Send + Sync + 'static + Hash + Eq + PartialEq {}
pub trait Value: Send + Sync + 'static {}

impl<T: Send + Sync + 'static + std::hash::Hash + Eq> Key for T {}
impl<T: Send + Sync + 'static> Value for T {}

// TODO(MrCroxx): use `expect` after `lint_reasons` is stable.
pub trait StorageKey: Key + Serialize + DeserializeOwned {}
impl<T> StorageKey for T where T: Key + Serialize + DeserializeOwned {}

// TODO(MrCroxx): use `expect` after `lint_reasons` is stable.
pub trait StorageValue: Value + 'static + Serialize + DeserializeOwned {}
impl<T> StorageValue for T where T: Value + Serialize + DeserializeOwned {}

```